tests/test_dataset.py

```diff
@@ -164,6 +165,10 @@ def _export(dataset_name, task: TaskType, root_dir):
     if task in [TaskType.OBJECT_DETECTION, TaskType.INSTANCE_SEGMENTATION, TaskType.CLASSIFICATION]:
         dataset.export("coco")
     if task in [TaskType.OBJECT_DETECTION, TaskType.INSTANCE_SEGMENTATION, TaskType.CLASSIFICATION]:
+        if task == TaskType.CLASSIFICATION:
+            train_ids = load_json(dataset.train_set_file)
+            train_ids.extend([1, 4])  # for yolo classification export test
```

Comment:
What is this for?

Reply:
To ensure that there is at least one annotation in each category during testing.